coalib/bearlib/languages/definitions/Ruby.py

```diff
@@ -3,4 +3,10 @@
 
 @Language
 class Ruby:
-    pass
+    aliases = 'rb',
+
```

Comment:
Why is this blank line here?

Reply:
I use the python as a template.  i also saw it in cpp, and I though it was for readability. So i kept it.
Is it bad?

Reply:
I think you can remove this (as I saw in other templates).

Reply:
I dont know why it is in the Python one.

IMO it isnt very useful.